airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/commands.py

```diff
@@ -152,9 +152,11 @@ def should_use_remote_secrets(use_remote_secrets: Optional[bool]) -> bool:
         "bump_version": "pipelines.airbyte_ci.connectors.bump_version.commands.bump_version",
         "migrate_to_base_image": "pipelines.airbyte_ci.connectors.migrate_to_base_image.commands.migrate_to_base_image",
         "migrate-to-poetry": "pipelines.airbyte_ci.connectors.migrate_to_poetry.commands.migrate_to_poetry",
```

Comment:
@alafanechere nit: can you make an issue and rename `migrate-to-poetry` here to snake_case so it's consistent with everything else?